### PR TITLE
Add the knative 0.15.1 releases into kodata

### PR DESCRIPTION
**Pull request by houshengbo**
## Proposed Changes

* This PR adds the manifests of 0.15.1 into kodata.

### CODE REVIEW

**Comment by matzew** [comment]
@houshengbo  will there be a 0.15.x release of the opeartor?  If so, will the `release 0.15` branch be updated ? :thinking:

**Comment by houshengbo** [comment]
yes. this will be backported into release 0.15.

File: cmd/operator/kodata/knative-serving/0.15.1/1-serving-crds.yaml [hunk 1]
```diff
@@ -0,0 +1,602 @@
+# Copyright 2019 The Knative Authors
```
Comment (Cynocracy) [inline cmd/operator/kodata/knative-serving/0.15.1/1-serving-crds.yaml:1]: Should we just always apply the CRDs early in the Operator logic?

File: cmd/operator/kodata/knative-serving/0.15.1/5-serving-storage-version-migration.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# Copyright 2020 The Knative Authors
```
Comment (Cynocracy) [inline cmd/operator/kodata/knative-serving/0.15.1/5-serving-storage-version-migration.yaml:1]: What's the difference between this and 3-serving-storage-version-migration?

Comment (houshengbo) [reply cmd/operator/kodata/knative-serving/0.15.1/5-serving-storage-version-migration.yaml:1]: I do not see there is a `3-serving-storage-version-migration`.

File: cmd/operator/kodata/knative-serving/0.15.1/1-serving-crds.yaml (hunk 1 above)
Comment (houshengbo) [reply cmd/operator/kodata/knative-serving/0.15.1/1-serving-crds.yaml:1]: We can do that in the reconciler, but reconciler needs to look up the CRD by the label `knative.dev/crd-install: "true"`.

**Pull request merged**